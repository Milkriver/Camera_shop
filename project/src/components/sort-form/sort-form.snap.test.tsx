import {render} from '@testing-library/react';
import { createMemoryHistory } from 'history';
import HistoryRouter from '../history-router/history-router';
import SortForm from './sort-form';

const history = createMemoryHistory();

describe('Component: SortForm', () => {
  it('should render correctly', () => {
    const {container} = render(
        <HistoryRouter history={history}>
        <SortForm />
      </HistoryRouter>
    );

    expect(container).toMatchSnapshot();
  });
});