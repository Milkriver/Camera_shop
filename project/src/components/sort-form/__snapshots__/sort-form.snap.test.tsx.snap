// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: SortForm should render correctly 1`] = `
<div>
  <div
    class="catalog-sort"
  >
    <form
      action="#"
    >
      <div
        class="catalog-sort__inner"
      >
        <p
          class="title title--h5"
        >
          Сортировать:
        </p>
        <div
          class="catalog-sort__type"
        >
          <div
            class="catalog-sort__btn-text"
          >
            <input
              id="price"
              name="sort"
              type="radio"
              value="price"
            />
            <label
              for="price"
            >
              по цене
            </label>
          </div>
          <div
            class="catalog-sort__btn-text"
          >
            <input
              id="rating"
              name="sort"
              type="radio"
              value="rating"
            />
            <label
              for="rating"
            >
              по популярности
            </label>
          </div>
        </div>
        <div
          class="catalog-sort__order"
        >
          <div
            class="catalog-sort__btn catalog-sort__btn--up"
          >
            <input
              aria-label="по возрастанию"
              id="up"
              name="sort-icon"
              type="radio"
              value="asc"
            />
            <label
              for="up"
            >
              <svg
                aria-hidden="true"
                height="14"
                width="16"
              >
                <use
                  xlink:href="#icon-sort"
                />
              </svg>
            </label>
          </div>
          <div
            class="catalog-sort__btn catalog-sort__btn--down"
          >
            <input
              aria-label="по убыванию"
              id="down"
              name="sort-icon"
              type="radio"
              value="desc"
            />
            <label
              for="down"
            >
              <svg
                aria-hidden="true"
                height="14"
                width="16"
              >
                <use
                  xlink:href="#icon-sort"
                />
              </svg>
            </label>
          </div>
        </div>
      </div>
    </form>
  </div>
</div>
`;
